<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReaderMainPresenter</class>
 <widget class="QMainWindow" name="ReaderMainPresenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>771</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTableWidget" name="BooksTable"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>371</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="1">
      <widget class="QLineEdit" name="AuthorText"/>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="ByNameBookButton">
       <property name="text">
        <string>by name of book</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Choose type of books:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="BookNameText"/>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="ByAuthorButton">
       <property name="text">
        <string>by author of book</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="ByAddressButton">
       <property name="text">
        <string>by address of library</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="ByNameLibraryButton">
       <property name="text">
        <string>by name of library</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QRadioButton" name="FreeBooksButton">
         <property name="text">
          <string>free books</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="MyBooksButton">
         <property name="text">
          <string>my books</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="AddressText"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="LibraryNameText"/>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Choose filters for books:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="FindButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>210</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>find!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
